go-controller/hybrid-overlay/pkg/controller/node_linux.go

```diff
@@ -42,7 +42,10 @@ type flowCacheEntry struct {
 // NodeController is the node hybrid overlay controller
 type NodeController struct {
 	nodeName string
-	// an atomic uint32 for testing purposes 0 = uninitialized and 1 = initialized
+	// an atomic uint32 to check the initialization status
+	// 0 = not initialized
+	// 1 = drIP and drMAC are set but the initial pods are not wired for hybrid overlay
+	// 2 = drIP and drMAC are set and the intiial pods are wired for hybrid overlay
```

Comment:
`initial`

Reply:
Nice spot, however, this comment does not exist anymore so I am marking this conversation as done.